modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3504,6 +3507,48 @@ def groupby_cov(
             drop=drop,
         )
 
+    def groupby_rolling(
+        self,
+        by,
+        agg_func,
+        axis,
+        groupby_kwargs,
+        rolling_kwargs,
+        agg_args,
+        agg_kwargs,
+        drop=False,
+    ):
+        unsupported_groupby = (
+            agg_func in ("corr", "cov") or rolling_kwargs.get("on") is not None
```

Comment:
Why are "corr" and "cov" not supported?

Reply:
'corr' and 'cov' require knowledge about the whole row axis (all columns have to be available in the same partitions), this requirement is not being satisfied in the current groupby implementation

Reply:
I would appreciate a comment in the code about this.

Reply:
@dchigarev many thanks